quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ClusterConfigKeepAliveDistTest.java

```diff
@@ -60,20 +70,97 @@ void testMaxCountApplied(KeycloakDistribution dist) {
         String args = sb.toString();
         dist.run(args.split(" "));
 
-        ParserRegistry parserRegistry = new ParserRegistry();
         for (String cache : maxCountCaches) {
-            String configJson = when()
-                  .get("/realms/master/test-resources/cache/" + cache + "/config")
-                  .thenReturn()
-                  .getBody()
-                  .jsonPath()
-                  .prettyPrint();
-
-            ConfigurationBuilderHolder configHolder = parserRegistry.parse(configJson, MediaType.APPLICATION_JSON);
-            // Workaround for ISPN-16595
-            String cacheName = CaseFormat.LOWER_CAMEL.to(CaseFormat.LOWER_HYPHEN, cache);
-            Configuration config = configHolder.getNamedConfigurationBuilders().get(cacheName).build();
+            Configuration config = getCacheConfiguration(cache);
             assertEquals(maxCount, config.memory().maxCount());
         }
     }
+
+    @Test
+    @TestProvider(TestRealmResourceTestProvider.class)
+    void testNumOwnersWithPersistentSessions(KeycloakDistribution dist) {
+        doNumOwnerTest(dist, false);
+    }
+
+    @Test
+    @TestProvider(TestRealmResourceTestProvider.class)
+    void testNumOwnersWithVolatileSessions(KeycloakDistribution dist) {
+        doNumOwnerTest(dist, true);
+    }
+
+    @Test
+    @TestProvider(TestRealmResourceTestProvider.class)
+    void testCheckMinimumNumOwners(KeycloakDistribution dist) {
+        Set<String> caches = Set.of(
```

Comment:
Can't we use `InfinispanConnectionProvider.CLUSTERED_CACHE_NUM_OWNERS` here?

Reply:
Yes, we can. The `CLUSTERED_CACHE_NUM_OWNERS` was added at a later stage, and I forgot to update this method :sweat_smile: 
Fixed!